CONTRIBUTING.md

```diff
@@ -71,6 +71,20 @@ When afraid to touch force and risk losing your work (do backups!..),
 try *merging master into your branch* instead of rebasing onto it.
 This is discouraged, as it produces ugly hard-to-maintain commit history.
 
+## Commit message rules
+
+When commiting your changes into the repository you may want to follow some
+rules to make the git history more readable and consistent:
+
+* Start the message capitalized (first letter in uppercase)
+* Be short and concise, the whole concept must fit one line
+* If a command is inlined, use backticks
+* Add a double-hashtag if the change matters for the changelog
+* Search the changelog to known hashtags (see sys/release-notes.sh)
```

Comment:
I don't think this entry is necessary.

Reply:
Maybe reword to: hashtags are used by the release notes script

Reply:
I'm not sure new contributors would care to know.

Reply:
It's wrong to ask new developers to search something. The all available hashtags should be listed here, in this document, otherwise they can't be made mandatory.

Reply:
Feel free to commit a list of tags or do you prefer a shell oneliner that prints them all? This pr is suposed to be collaborative. So feel free to submit suggestion changes